core/vm/gas.go

```diff
@@ -52,3 +54,14 @@ func callGas(isEip150 bool, availableGas, base uint64, callCost *uint256.Int) (u
 
 	return callCost.Uint64(), nil
 }
+
+// constantMultiGas returns the constant multi-gas cost of an opcode.
+func constantMultiGas(cost uint64, op OpCode) *multigas.MultiGas {
+	if op == SELFDESTRUCT && cost == params.SelfdestructGasEIP150 {
```

Comment:
Can we have some commentary about why SELFDESTRUCT is special?
Maybe it would be better-suited to PR description, or updating the ADR-002 with information that will capture the answer to the question "why?" for posterity?

Reply:
I was following the tracer here, but in retrospect, it is not clear why the base cost of self-destruct is multi-dimensional instead of just being computation.

Reply:
Sorry. I'm not saying that it should't be multi-dimensional. I'm guessing that Tyler and Ben thought this through, and that's why the tracer sets it up that way. But, can we write down either directly as inline documentation comments or as part of ARD-002.md the rationale so we don't later forget why it's special cased?

Reply:
After reviewing the opcode it makes sense to be multi-dimensional. I will add a comment.